Aquality.Selenium.Core/src/Aquality.Selenium.Core/Waitings/ConditionalWait.cs

```diff
@@ -68,18 +68,15 @@ public T WaitFor<T>(Func<IWebDriver, T> condition, TimeSpan? timeout = null, Tim
         /// <param name="condition">Predicate for waiting</param>
         /// <param name="timeout">Condition timeout. Default value is <see cref="ITimeoutConfiguration.Condition"/></param>
         /// <param name="pollingInterval">Condition check interval. Default value is <see cref="ITimeoutConfiguration.PollingInterval"/></param>
+        /// <param name="exceptionsToIgnore">Possible exceptions that have to be ignored. </param>
         /// <returns>True if condition satisfied and false otherwise.</returns>
```

Comment:
I think documentation could be removed, it already exists for interface.

Reply:
yes it is, but if it will remain only in interface you won't be able to view it directly when you're viewing the ConditionWait class